contracts/mockup/VestingRegistryLogicMockUp.sol

```diff
@@ -6,4 +6,12 @@ contract VestingRegistryLogicMockup is VestingRegistryLogic {
     function isVestingAdress(address _vestingAddress) external view returns (bool isVestingAddr) {
         return true;
     }
+
+    function setTeamVesting(address _vesting, uint256 _vestingCreationType) external {
+        vestingCreationAndTypes[_vesting] = VestingCreationAndTypeDetail(
+            true,
+            uint120(VestingType.TeamVesting),
+            uint128(_vestingCreationType)
```

Comment:
named assignment would be more clear { var1: val1, var2: val2 ...}

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)